maintenance/wikia/VideoHandlers/nfcNormalizeVideos.php

```diff
@@ -1,6 +1,5 @@
 <?php
 ini_set('display_errors', 'stderr');
-putenv("SERVER_ID=177");
```

Comment:
Yah, I should have noticed this.  Both of these things (SERVER_ID and errors) are handled by the script that's actually invoked on the command line, runOnCluster.php